### PR TITLE
Session persistence

### PR DESCRIPTION
**Pull request by glossopm**
- Added in session persistence to the login functionality
- Redirects to homepage when navigating to the login page once logged in

Note: Since there is no logout feature, you need to delete cookies to logout at this time.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": false
```
Comment (sorsby) [inline .vscode/settings.json]: Add this file to .gitignore and remove from the repository.

File: globals/globals.py [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 
 app = Flask(__name__)
+app.config.from_object(os.environ['APP_SETTINGS'])
+print("############# CURRENT CONFIG: " + os.environ['APP_SETTINGS'] + " #############")
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+bcrypt = Bcrypt(app)
+login_manager = LoginManager(app)
+#login_manager.login_view = 'login'
+#login_manager.login_message_category = 'info'
```
Comment (sorsby) [inline globals/globals.py]: Remove these comments.

File: .vscode/settings.json (hunk 1 above)
Comment (sorsby) [inline .vscode/settings.json]: Add this file to .gitignore and remove from the repository.

Comment (sorsby) [inline .vscode/settings.json]: Add this file to .gitignore and remove from the repository.

File: globals/globals.py (hunk 2 above)
Comment (sorsby) [inline globals/globals.py]: Remove these comments.

File: globals/templates/base.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="{{ url_for('static', filename='styles.css') }}">
+    </head>
+
+    <nav class="navbar navbar-expand-lg navbar-light bg-transparent">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+    </button>
+    <a class="navbar-brand" href="/">Home</a>
+        <div class="collapse navbar-collapse" id="navbarTogglerDemo03">
```
Comment (sorsby) [inline globals/templates/base.html]: id=navbarTogglerDemo03 what does this do?

File: manage.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from flask_script import Manager
```
Comment (sorsby) [inline manage.py]: I don't think we need this file in this repository. We can run all the migrations from the api project.

**Review by sorsby — DISMISSED** [review]
See my comments.

**Review by sorsby — CHANGES_REQUESTED** [review]
See comments.

File: globals/templates/base.html (hunk 3 above)
Comment (glossopm) [reply globals/templates/base.html]: It uses bootstrap templates for a togglable navbar.

File: manage.py (hunk 4 above)
Comment (glossopm) [reply manage.py]: I've removed this as after review this is only necessary in one repo.

**Review by sorsby — APPROVED** [review]

**Pull request merged**